src/main/org/codehaus/groovy/runtime/DefaultGroovyMethods.java

```diff
@@ -8913,11 +8913,15 @@ public int compare(Map.Entry<K, V> e1, Map.Entry<K, V> e2) {
     }
 
     /**
-     * Removes the last item from the List. Using add() and pop()
-     * is similar to push and pop on a Stack.
-     * <pre class="groovyTestCase">def list = ["a", false, 2]
-     * assert list.pop() == 2
-     * assert list == ["a", false]</pre>
+     * Removes the initial item from the List, similar to pop on a Stack.
```

Comment:
I wonder if most associate removing items from the top of a Stack, so in this case would it make sense to say 
>..similar to pop on a Deque.